apps/frontend/src/index.tsx

```diff
@@ -20,27 +21,34 @@ createRoot(document.querySelector("#root") as HTMLElement).render(
 			<RouterProvider
 				routes={[
 					{
-						element: (
-							<ProtectedRoute>
-								<App />
-							</ProtectedRoute>
-						),
+						children: [
+							{
+								element: (
+									<ProtectedRoute>
+										<DefaultPage />
```

Comment:
Let's name it Projects
```suggestion
										<Projects />
```